src/test/resources/log4j2-spring.properties

```diff
@@ -13,8 +13,8 @@ appender.console.layout.pattern = [%d{dd/MMM/yyyy HH:mm:ss}] [%X{OHUserGroup}:%X
 # File Appender (with classes), daily rotation
 appender.rolling.type = RollingFile
 appender.rolling.name = RollingFile
-appender.rolling.fileName= LOG_DEST/openhospital.log
-appender.rolling.filePattern= LOG_DEST/openhospital.log.%d{yyyy-MM-dd}
+appender.rolling.fileName= logs/openhospital.log
+appender.rolling.filePattern= logs/openhospital.log.%d{yyyy-MM-dd}
```

Comment:
you sure about this change?   on my system the log files are found under `LOG_DEST`

Reply:
LOG_DEST is a placeholder, one should replace it with a folder of choise, not a must change actually...